src/Curiosity/Runtime.hs

```diff
@@ -268,7 +273,14 @@ handleCommand runtime@Runtime {..} display user command = do
               let value' = if useHs
                     then show value
                     else LT.toStrict (Aeson.encodeToLazyText value)
-              display value'
+              if short
```

Comment:
I wouldn't use a `bool` for `short` displays directly, but a sum-type. 
``` haskell 
data DisplayType = Normal | Short deriving (Eq, Show) 
```
This can be parsed with the `A.flag` and is more expressive. Also, we don't run into _boolean blindness_.

Reply:
I've filled an issue: https://github.com/hypered/curiosity/issues/59